chain/wallet/multi.go

```diff
@@ -152,6 +152,17 @@ func (m MultiWallet) WalletImport(ctx context.Context, info *types.KeyInfo) (add
 	return w.WalletImport(ctx, info)
 }
 
+func (m MultiWallet) WalletMsigImport(ctx context.Context, idAddress address.Address, robustAddress address.Address) error {
+	var local getif = m.Local
+
```

Comment:
there should be a check to make sure that whoever is importing the msig has the key to one of the msig signers, because later we also have to pick which of the signer addresses to use for msig operations

Reply:
创建多签地址，创建人并不一定作为多签地址的签名人哦。
另外考虑到多签地址跟签名者没有强关联关系。
这个修改可能会导致原创建接口进行相应的修改。
当然我觉得统一修改也会更简化使用者的操作，您可以再考虑一下，需不需要优化